README.md

```diff
@@ -158,13 +170,18 @@ When ofc-bootstrap has completed and you know the IP of your LoadBalancer:
 * `auth.system.domain`
 * `*.domain`
 
-#### Configure the GitHub App webhook
+#### Configure the GitHub / GitLab App webhook
 
-Now over on GitHub enter the URL for webhooks:
+Now over on GitHub / GitLab enter the URL for webhooks:
```

Comment:
@martindekov do we explain in either here or the main repo how to obtain all the config and settings for GitLab?

Reply:
Sorry Alex I saw that comment just now .. otherwise yes in [here](https://github.com/openfaas/openfaas-cloud/blob/master/docs/GITLAB.md)

Reply:
@ivanayov please add Martin's link above to this PR asap. 
https://github.com/openfaas/openfaas-cloud/blob/master/docs/GITLAB.md

Reply:
@alexellis It's added now below